lib/pdk/report.rb

```diff
@@ -78,7 +78,7 @@ def to_junit(target = self.class.default_target)
     def to_text(target = self.class.default_target)
       events.each do |_tool, tool_events|
         tool_events.each do |event|
-          target.puts("\n" << event.to_text) unless event.pass?
+          target.puts(event.to_text) unless event.pass?
```

Comment:
This just undoes a change from the previous commit. If you `rebase --interactive`, and remove the change from the first commit, you can get rid of it here too.